fix(ivy): correctly project bare ICU expressions

ocombe opened the pull request:
Projecting bare ICU expressions failed because we would assume that component's content nodes would be projected later and doing so at that point would be wasteful. But ICU nodes are handled independently and should be inserted immediately because they will be ignored by projections.

FW-1348 #resolve

AndrewKushnir reviewed (approved):
LGTM 👍 On nit: it might be helpful to have a comment in the code  (above updated condition), similar to what you wrote in PR description.

AndrewKushnir commented:
[Presubmit](http://test/OCL:250305494:BASE:250305505:1559062779413:5c46b7be)

AndrewKushnir reviewed (approved):
LGTM 👍

AndrewKushnir commented:
@ocombe could you plz look into `integration_test` failure (size limit needs to be updated). Thank you.

ocombe committed:
fix(ivy): correctly project bare ICU expressions

Projecting bare ICU expressions failed because we would assume that component's content nodes would be projected later and doing so at that point would be wasteful. But ICU nodes are handled independently and should be inserted immediately because they will be ignored by projections.

FW-1348 #resolve

ocombe committed:
fixup! fix(ivy): correctly project bare ICU expressions

mhevery commented:
http://test/OCL:251236135:BASE:251236170:1559575531264:9be726a8
